test-site/gatsby-config.js

```diff
@@ -36,25 +36,23 @@ const wpPluginOptions = !process.env.DEFAULT_PLUGIN_OPTIONS
         },
       },
       type: {
-        Cart: {
-          exclude: true,
-        },
         TypeLimitTest: {
           limit: 1,
         },
         TypeLimit0Test: {
           limit: 0,
         },
-        TranslationFilterTest: {
-          where: `language: FR`,
-          beforeChangeNode: async ({ remoteNode, type }) => {
-            if (remoteNode.language && remoteNode.language.slug !== `fr`) {
-              return {
-                cancelUpdate: true,
-              }
-            }
-          },
-        },
+        // TODO: test translation support using our own test wp graphql plugin
```

Comment:
This was actually to test our where and beforeChangeNode type options :)

Reply:
ohh i see, should i just delete those fields then?

Reply:
Maybe update the todo message so we remember to test that later 👍